### DISCUSSION TITLE
How to change settings for "Do not allow bypassing the above settings"

### DISCUSSION THREAD

**Discussion by mirzaprangon** [integrations/terraform-provider-github#1493] [Q&A]
Hello
In the UI of branch protection rule I can change a setting called the "Do not allow bypassing the above settings". How do I set or change this settings using terraform? I could not find any document on this.

![image](https://user-images.githubusercontent.com/38160503/213074793-06220040-0f7d-49cd-a8f5-fc0d210edc6a.png)

**Comment by johnheusinger** [comment]
I agree that the documentation for this could be better. The setting corresponds to [enforce_admins](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/branch_protection#enforce_admins) in the `github_branch_protection` resource.
